### PR TITLE
Axios request

### PR DESCRIPTION
**Pull request by claire-melbourne**
small PR for public/helpers/getSiteData axios request invoked in public/src/index.jsx

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,5 +22,5 @@ coverage
 
 # Compiled Dirs (http://nodejs.org/api/addons.html)
 build/
-bundle.js
+bundle2.js
```
Comment (owenwexler) [inline .gitignore:25]: bundle2?  did the name of the bundle generated by Webpack change?

**Review by owenwexler — APPROVED** [review]
Other than the mysterious name change of the bundle, which I commented on, LGTM.

**Review by owenwexler — APPROVED** [review]
LGTM!

File: .gitignore (hunk 1 above)
Comment (claire-melbourne) [reply .gitignore:25]: I am working on this!

**Pull request merged**